docs/content/guide/interpolation.ngdoc

```diff
@@ -0,0 +1,125 @@
+@ngdoc overview
+@name Interpolation
+@sortOrder 275
+@description
+
+Interpolation markup with embedded @link {guide/expressions expressions} is used by Angular to
+provide data-binding to text nodes and attribute values.
+
+An example of interpolation is shown below:
+
+```html
+<a ng-href="img/{{username}}.jpg">Hello {{username}}!</a>
+```
+
+### How text and attribute bindings work
+
+During the compilation process the {@link ng.$compile compiler} uses the {@link ng.$interpolate $interpolate}
+service to see if text nodes and element attributes contain interpolation markup with embedded expressions.
+If that is the case, the compiler adds an interpolationDirective to the node and
+registers {@link ng.$rootScope.Scope#$watch watches} on the computed interpolation function,
+which will update the corresponding text nodes or attribute values as part of the
+normal {@link ng.$rootScope.Scope#$digest digest} cycle.
+
+### Binding to boolean attributes
+
+Attributes such as `disabled` are called `boolean` attributes, because their presence means `true` and
+their absence means `false`. We cannot use normal attribute bindings with them, because the HTML
+specification does not require browsers to preserve the values of boolean attributes. This means that
+if we put an Angular interpolation expression into such an attribute then the binding information
+would be lost, because the browser ignores the attribute value.
+
+In the following example, the interpolation information would be ignored and the browser would simply
+interpret the attribute as present, meaning that the button would always be disabled.
+
+ ```html
+  <div ng-init="isDisabled = false">
+    <button disabled="{{isDisabled}}">Disabled</button>
+  </div>
+```
+
+For this reason, Angular provides special `ng`-prefixed directives for the following boolean attributes:
+{@link ngDisabled `disabled`}, [@link ngSelected `selected`}, {@link ngChecked `checked`},
+{@link ngReadonly `readOnly`} , and [@link ngOpen `open`}.
```

Comment:
ngMultiple doesn't exist actually. At least not in attrs.js (also no docs). I think this is because binding to multiple in select is not supported, and in inputs it's not yet supported.

+1 for ngRequired